### PR TITLE
Profile Addition

### PR DESCRIPTION
**Pull request by YashkShrivas4491**
## Changes proposed

 I edited my profile addition and make the PR.

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3872"></a>

### CODE REVIEW

File: public/data/Obito9596.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (Panquesito7) [inline public/data/Obito9596.json:1]: I don't think you should add a profile for others. 🤔

Comment (YashkShrivas4491) [reply public/data/Obito9596.json:1]: this is my profile earlier, i changed the name to YashKumarShrivas right now, that's why along with new pr the old one also added. 
Obito9596-->YashkShrivas4491
sorry for the inconvenience caused by me.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Your filename must match your GitHub username.
Please change/update it accordingly. Any help, let us know! 🙂

**Comment by YashkShrivas4491** [comment]
> Your filename must match your GitHub username. Please change/update it accordingly. Any help, let us know! 🙂

Thanks for addressing me that issue.

**Pull request closed without merging**